backend/README.md

```diff
@@ -37,3 +63,12 @@ cd backend
 dotnet format --severity info
 ```
   
+## Configuration  
+
+The project has two [appsettings](https://docs.microsoft.com/en-us/iis-administration/configuration/appsettings.json)
+files.  
+The base `appsettings.json` file is for common variables across all environments, while the
+`appsetings.Development.json` file is for variables specific to the Dev environments, such as the client ID's for the
+various app registrations used in development.
+
+In the future, most of the config should probably be pulled from various Azure Key Vaults, one for each environment.
```

Comment:
Unnecessary sentence? Rather make an issue than adding TODOs in README

Reply:
Agreed